vignettes/release_strategy.Rmd

```diff
@@ -39,34 +39,58 @@ updates via the community meetings, especially any anticipated breaking changes.
 
 # Package Release Process
 
-## Quarterly Release
-
-A package release is done in four parts:
-
-1) Verify that all CI/CD checks and then bundle up and send off to CRAN. See the [chapter](https://r-pkgs.org/release.html#decide-the-release-type) in R Packages for more details.
+## Planned Release
+
+A package release is done in the following steps:
+
+1) Create the release version:
+    a) Create a new feature branch from `main`.
+    a) Update the `Versions:` field in the `DESCRIPTION` file.
+    a) Update `NEWS.md` (replace "(development version)" with the version to be
+    released)
+    a) Create a PR into `main` and merge it. The title of the PR must contain "[skip vbump]".
+    Otherwise, the vbump workflow sets the version to a development version and
+    CRAN will reject the submission.
+1) Bundle up (from `main`) and send off to CRAN. See the [chapter](https://r-pkgs.org/release.html#decide-the-release-type) in R Packages for more details.
 1) If CRAN asks for modifications, repeat steps 1-2 as necessary.
 1) Once the package is accepted and available on CRAN, a [GitHub action](https://github.com/pharmaverse/admiral/actions/workflows/pages/pages-build-deployment) is set up to rebuild the `{admiral}` website with all the updates for this release.
```

Comment:
@ddsjoberg , what is meant by "is set up"? Is this done automatically or does it require an action from the developer?

Reply:
I don't think there is anything new that is set up. Whenever we make an admiral release, the admiral website is built. I am not aware of anything else occuring.

Reply:
OK, I have combined this step and the following one. Does this make sense?